src/plugins/multifile/README.md

```diff
@@ -102,7 +102,7 @@ kdb ls user/tests/multifile
 
 kdb rm -r user/tests/multifile/test.ini
 
-stat ~/.config/multifile/test.ini
+stat $(dirname $(kdb file user))/multifile/test.ini
```

Comment:
We also should add build jobs which use non-standard config paths (ideally also with some umlauts, white spaces, ...)

Reply:
AFAIK most if not all build jobs (certainly the Jenkins ones) use non-standard config paths inside the workspace directory. I also run all local tests with a similar setup.

About umlauts/white-space I am not sure, we might want to create an issue

Reply:
So the multifile test cases are not running on our build server?

Reply:
I just checked and at least the `debian-stable-full` build runs this test successfully... Maybe some of the changes to `kdbGet` I made, brought up a hidden bug which was suppressed before

Reply:
Thank you for looking into it!